documentation/modules/ROOT/pages/operations/debezium-platform.adoc

```diff
@@ -508,46 +553,63 @@ The pipeline section is the place where you connect the "dots". You can define w
 [.responsive]
 video::cY7TS0nWBoY[youtube, title="Create, edit and remove a pipeline"]
 
-==== Creating pipelines
+==== Creating pipeline
 
-. From the platform UI, open the **Pipeline** menu and click **Create your first pipeline**  to open the **Pipeline Designer**.
-+
-From the **Pipeline Designer** you can add the pieces for your data pipeline.
-. Click the **+ Source** box to add a source, and then choose a previously created source, or create a new source.
+From the platform UI, open the **Pipeline** menu and click **Create your first pipeline** to open the **Pipeline Designer**.
+From the Pipeline Designer you can add the pieces for your data pipeline.
 
-Click the **+ Destination** box to add a destination .
+. Click the **+ Source** box to add a source, and then choose a previously created source, or create a new source.
 
 . (Optional) Click the **+ Transform** box to apply one or more transformations.
 
+. Click the **+ Destination** box to add a destination,and then choose a previously created destination, or create a new destination.
+
+The order in which a Transform is applied to the source is important and decides the final output at the end, so you have to be mindfully of the order in which you add transforms.
+
+In the pipeline designer you have the ability to delete a transform or alter the sequency of applied transforms. For more information about using the pipeline designer, see xref:debezium-platform-using-the-pipeline-designer-to-remove-and-order-transformations[using the Pipeline designer to remove and order transformations]
+
+[NOTE]
 Transformations that are configured with a predicate are marked with a predicate icon (image:predicate-icon.png[Transformation configured with a predicate]).
-A tooltip shows the name of the predicate.
+A tooltip shows the name of the predicate type.
+
+image::debezium-platform-pipeline.png[{prodname} platform pipeline]
 
 After you finish designing your pipeline, click **Configure Pipeline**, and then specify the name, description, and logging level for the pipeline.
 
+
+==== Creating pipeline using {prodname} Server config
+
+Pipeline Designer also provides you the option to upload the {prodname} Server configuration properties file directly to create the data pipeline resources i.e source, destination and transforms.
+
+In the pipeline designer click on **DBZ server config** which will open a modal where you can upload or drag and drop your {prodname} server properties file. Once you upload the file click on **Create** button.
```

Comment:
```
*DBZ server config*
```
Is that correct? Or should it read *{prodname} server config(uration)*?  I don't see a PNG file that shows the actual UI.